ingest/src/main/java/com/connexta/ingest/config/ApplicationConfiguration.java

```diff
@@ -18,8 +20,27 @@
 @EnableAutoConfiguration
```

Comment:
This annotation tries to guess what beans the application needs. I think this is where Eric said we should be explicit.

Reply:
Should every bean be named? Or just where we have more than one bean of the same type?

Reply:
I removed `@EnableAutoConfiguration`.

Reply:
Right now, we are only naming beans with the same type.

Reply:
Eric said using `@EnableAutoConfiguration` is fine. We need it to create the `RestTemplateBuilder`.